src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -686,6 +686,8 @@ export function updateDeprecatedFeaturesIntoOptions(userFeatures: DevContainerFe
 
 export async function getFeatureIdType(params: CommonParams, userFeatureId: string, lockfile: Lockfile | undefined) {
 	const { output } = params;
+	const error_message = `Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements.
```

Comment:
Can we move this variable to L703 ie inside [this](https://github.com/devcontainers/cli/pull/838/files#diff-93622e1db0f140d14a85a65ca0a4522090b385c3964de02011229c0523095470R702) condition?

Reply:
done